Sources/ZcashLightClientKit/Synchronizer/SDKSynchronizer.swift

```diff
@@ -766,9 +769,13 @@ public class SDKSynchronizer: Synchronizer {
             for service in services {
                 group.addTask {
                     let startTime = Date().timeIntervalSince1970
-                    let info = try? await service.service.getInfo()
+                    // ServiceMode to resolve
+                    // called when performance of servers is evaluated
+                    let info = try? await service.service.getInfo(mode: .defaultTor)
```

Comment:
These should be individually grouped per trialled service, and not linked to the main sync group:
```suggestion
                    let mode = .torInGroup("SDKSynchronizer.evaluateBestOf(\(service.originalEndpoint))")
                    let info = try? await service.service.getInfo(mode: mode)
```

Reply:
Updated